relayer/src/config.rs

```diff
@@ -78,11 +89,28 @@ pub struct ChainConfig {
     pub client_ids: Vec<String>,
     #[serde(default = "default::gas")]
     pub gas: u64,
+    #[serde(default = "default::clock_drift", with = "humantime_serde")]
+    pub clock_drift: Duration,
     #[serde(default = "default::trusting_period", with = "humantime_serde")]
     pub trusting_period: Duration,
+    #[serde(default)]
+    pub trust_threshold: TrustThreshold,
```

Comment:
Should this be in the client config? Same question for the `trusting_period`? Different apps may use different clients of same chain if they have different security requirements.

Reply:
@ebuchman ^

Reply:
Tracked in #356